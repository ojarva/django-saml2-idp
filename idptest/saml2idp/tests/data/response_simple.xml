<samlp:Response xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion" xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol" Destination="https://www.google.com/a/abrickaday.com/acs" ID="_2972e82c07bb5453956cc11fb19cad97ed26ff8bb4" InResponseTo="mpjibjdppiodcpciaefmdahiipjpcghdcfjodkbi" IssueInstant="2011-08-11T23:38:34Z" Version="2.0">
    <samlp:Status>
        <samlp:StatusCode Value="urn:oasis:names:tc:SAML:2.0:status:Success"></samlp:StatusCode>
    </samlp:Status>
    <saml:Assertion xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ID="_7ccdda8bc6b328570c03b218d7521772998da45374" IssueInstant="2011-08-11T23:38:34Z" Version="2.0">
        <saml:Subject>
            <saml:NameID Format="urn:oasis:names:tc:SAML:2.0:nameid-format:email" SPNameQualifier="google.com">student@abrickaday.com</saml:NameID>
            <saml:SubjectConfirmation Method="urn:oasis:names:tc:SAML:2.0:cm:bearer">
                <saml:SubjectConfirmationData InResponseTo="mpjibjdppiodcpciaefmdahiipjpcghdcfjodkbi" NotOnOrAfter="2011-08-11T23:43:34Z" Recipient="https://www.google.com/a/abrickaday.com/acs"></saml:SubjectConfirmationData>
            </saml:SubjectConfirmation>
        </saml:Subject>
        <saml:Conditions NotBefore="2011-08-11T23:38:04Z" NotOnOrAfter="2011-08-11T23:43:34Z">
            <saml:AudienceRestriction>
                <saml:Audience>google.com</saml:Audience>
            </saml:AudienceRestriction>
        </saml:Conditions>
        <saml:AuthnStatement AuthnInstant="2011-08-11T23:38:34Z" SessionIndex="_ee277dff4e2db138d25dfcea7ccdf1d1db9ddea3f5" SessionNotOnOrAfter="2011-08-12T07:38:34Z">
            <saml:AuthnContext>
                <saml:AuthnContextClassRef>urn:oasis:names:tc:SAML:2.0:ac:classes:Password</saml:AuthnContextClassRef>
            </saml:AuthnContext>
        </saml:AuthnStatement>
    </saml:Assertion>
</samlp:Response>