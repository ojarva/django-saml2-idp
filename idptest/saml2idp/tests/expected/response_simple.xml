<samlp:Response xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol"
                xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion"
                ID="_2972e82c07bb5453956cc11fb19cad97ed26ff8bb4"
                Version="2.0"
                IssueInstant="2011-08-11T23:38:34Z"
                Destination="https://www.google.com/a/abrickaday.com/acs"
                InResponseTo="mpjibjdppiodcpciaefmdahiipjpcghdcfjodkbi"
                >
    <samlp:Status>
        <samlp:StatusCode Value="urn:oasis:names:tc:SAML:2.0:status:Success" />
    </samlp:Status>
    <saml:Assertion xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xmlns:xs="http://www.w3.org/2001/XMLSchema"
                    ID="_7ccdda8bc6b328570c03b218d7521772998da45374"
                    Version="2.0"
                    IssueInstant="2011-08-11T23:38:34Z"
                    >
        <saml:Subject>
            <saml:NameID SPNameQualifier="google.com"
                         Format="urn:oasis:names:tc:SAML:2.0:nameid-format:email"
                         >student@abrickaday.com</saml:NameID>
            <saml:SubjectConfirmation Method="urn:oasis:names:tc:SAML:2.0:cm:bearer">
                <saml:SubjectConfirmationData NotOnOrAfter="2011-08-11T23:43:34Z"
                                              Recipient="https://www.google.com/a/abrickaday.com/acs"
                                              InResponseTo="mpjibjdppiodcpciaefmdahiipjpcghdcfjodkbi"
                                              />
            </saml:SubjectConfirmation>
        </saml:Subject>
        <saml:Conditions NotBefore="2011-08-11T23:38:04Z"
                         NotOnOrAfter="2011-08-11T23:43:34Z"
                         >
            <saml:AudienceRestriction>
                <saml:Audience>google.com</saml:Audience>
            </saml:AudienceRestriction>
        </saml:Conditions>
        <saml:AuthnStatement AuthnInstant="2011-08-11T23:38:34Z"
                             SessionNotOnOrAfter="2011-08-12T07:38:34Z"
                             SessionIndex="_ee277dff4e2db138d25dfcea7ccdf1d1db9ddea3f5"
                             >
            <saml:AuthnContext>
                <saml:AuthnContextClassRef>urn:oasis:names:tc:SAML:2.0:ac:classes:Password</saml:AuthnContextClassRef>
            </saml:AuthnContext>
        </saml:AuthnStatement>
    </saml:Assertion>
</samlp:Response>
